src/EA.Iws.Web/Areas/AdminExportAssessment/Controllers/OverviewController.cs

```diff
@@ -33,13 +33,15 @@ public async Task<ActionResult> Index(Guid id)
             var canAddProducer = Task.Run(() => authorizationService.AuthorizeActivity(ExportNotificationPermissions.CanAddProducer)).Result;
             var canAddRemoveTransitState = Task.Run(() => authorizationService.AuthorizeActivity(ExportNotificationPermissions.CanAddRemoveTransitState)).Result;
             var canEditContactDetails = Task.Run(() => authorizationService.AuthorizeActivity(ExportNotificationPermissions.CanEditContactDetails)).Result;
+            var canEditOperationCodes = Task.Run(() => authorizationService.AuthorizeActivity(ExportNotificationPermissions.CanEditOperationCodes)).Result;
 
             var model = new NotificationOverviewViewModel(result);
             model.AmountsAndDatesViewModel.CanChangeNumberOfShipments = canChangeNumberOfShipments;
             model.JourneyViewModel.CanChangeEntryExitPoint = canChangeEntryExitPoints;
             model.JourneyViewModel.CanAddRemoveTransitState = canAddRemoveTransitState && IsEditableStatus(result.SubmitSummaryData.Status);
             model.OrganisationsInvolvedViewModel.CanAddProducer = canAddProducer;
             model.OrganisationsInvolvedViewModel.CanEditContactDetails = canEditContactDetails;
+            model.RecoveryOperationViewModel.CanEditCodes = canEditOperationCodes;
```

Comment:
You also need to ensure that notification status is Consented as per A/C. You can re-use the `result.SubmitSummaryData.Status` property.